src/blueprints/tv2/factories/tv2-graphics-action-factory.ts

```diff
@@ -134,5 +148,162 @@ export class Tv2GraphicsActionFactory {
     }
   }
 
+  private createFullscreenGraphicActionsFromActionManifests(blueprintConfiguration: Tv2BlueprintConfiguration, actionManifests: Tv2GraphicActionManifest[]): PartAction[] {
+    switch (blueprintConfiguration.studio.GraphicsType) {
+      case Tv2GraphicsType.HTML: return this.createCasparCgFullscreen(blueprintConfiguration, actionManifests)
+      case Tv2GraphicsType.VIZ:
+      default: return this.createVizFullscreen(blueprintConfiguration, actionManifests)
+    }
```

Comment:
Personal opinion, but this is looks way more cluttered than:
```
if (blueprintConfiguration.studio.GraphicsType === Tv2GraphicsType.HTML) {
  return this.createCasparCgFullscreen(blueprintConfiguration, actionManifests)
}
return this.createVizFullscreen(blueprintConfiguration, actionManifests)
```

Reply:
How it is decided between which path to use has been updated since this comment.